introduction.md

```diff
@@ -2,6 +2,11 @@ Hola, mi nombre es Isaac Yriarte y estoy haciendo el curso de Code School Master
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?